README.md

```diff
@@ -18,13 +18,9 @@ microservices {
    }
 } 
 ```
-and bring in `payments-email-verification-cor` as a library dependency to your service **before v2.0.0**
-```
-"uk.gov.hmrc" %% "payments-email-verification-cor" % <version>
-```
-or for **v2.0.0 or later** choose one of the following based on whether your service is using Play 2.8 or Play 3.0
+and bring in `payments-email-verification-cor` as a library dependency to your service
```

Comment:
I meant for payments-email-verification to go to v3 so that you'd have to  bring in something like 
```
"uk.gov.hmrc" %% "payments-email-verirication-cor-play-30" %% "3.0.0"
``` 
The stuff in the README before would still apply when it's talking about before v2 and after v2 (but before v3).